src/common/options/global.yaml.in

```diff
@@ -5020,15 +5020,16 @@ options:
   desc: inject crc verification errors into bluestore device reads
   default: 0
   with_legacy: true
-- name: bluestore_debug_legacy_omap
-  type: bool
+- name: bluestore_omap_naming
+  type: str
   level: dev
-  desc: Allows mkfs to create OSD in legacy OMAP naming mode (neither per-pool nor per-pg).
+  desc: Allows to create omap in legacy mode
+  long_desc: Allows mkfs to create OSD in legacy OMAP naming mode.
     This is intended primarily for developers' purposes. The resulting OSD might/would
-    be transformed to the currrently default 'per-pg' format when BlueStore's quick-fix or
-    repair are applied.
-  default: false
-  with_legacy: true
+    be transformed to the currently default 'per-pg' format when BlueStore's quick-fix or
+    repair are applied. Acceptable values 'bulk', 'per-pool', 'per-pg'
```

Comment:
you might want to add "enum" clause, e.g.
  enum_values:
  - bulk
  - per-pool
  - per-pg